claat/main.go

```diff
@@ -188,6 +193,8 @@ Clicking on a directory representing an exported codelab will load
 all the required dependencies and render the generated codelab as
 it would appear in production.
 
+The serve command run first the build command.
```

Comment:
"The serve command always runs the build command first." or something like that. I'm not a native. @samtstern please correct.

Reply:
@x1ddos yeah your suggestion LGTM